include/readout/models/SourceEmulatorModel.hpp

```diff
@@ -181,6 +198,23 @@ class SourceEmulatorModel : public SourceEmulatorConcept
         // Fake timestamp
         payload.fake_timestamp(timestamp, m_time_tick_diff);
 
+        // Introducing frame errors
+
+        // This part can't be made generic, as frame header access is not uniform between frame types
```

Comment:
Hmm... I'm wondering if we should add a fake_errors() to the readout types. I don't like pulling in WIBFrame for generic models.